### PR TITLE
Add member image for Sravan Tallapaka

### LINKED ISSUE #329: Add member image for Sravan Tallapaka
adding an image of my cute ass

### PR DESCRIPTION
**Pull request by sravtall**
Closes #329

### CODE REVIEW

File: website/src/app/views/our-team/members.js [hunk 1]
```diff
@@ -239,6 +239,12 @@ const members = {
         linkedinURL: 'https://www.linkedin.com/in/subin-kim-bba754180/',
         imageURL: subin,
       },
+      {
```
Comment (alanchangxyz) [inline website/src/app/views/our-team/members.js:242]: alphabetical order (sr < su)

**Review by alanchangxyz — CHANGES_REQUESTED** [review]
just needs an ordering fix and if you end up coming to this later there may be merge conflicts with `2022-dev-bootcamp-task` (so pls fix them if github says there are any)

File: website/src/app/views/our-team/members.js (hunk 1 above)
Comment (sravtall) [reply website/src/app/views/our-team/members.js:285]: Should be fixed now unless I forgot the alphabet again.

**Review by alanchangxyz — APPROVED** [review]
lets go dude

**Pull request merged**